munit-scalacheck/shared/src/main/scala/munit/scalacheck/ScalaCheckSuite.scala

```diff
@@ -0,0 +1,39 @@
+package munit
+package scalacheck
+
+import org.scalacheck.Prop
+import org.scalacheck.{Test => ScalaCheckTest}
+import org.scalacheck.util.Pretty
+import scala.concurrent.Future
+
+trait ScalaCheckSuite extends FunSuite {
+
+  def property(
```

Comment:
Should we add a method overload that takes `property(name: String)`? For consistency with the test method. The goal is to avoid the "-language:implicitConversions` flag or `import scala.language.implicitConversions`